src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -63,6 +63,18 @@ describe("calcite-input-date-picker", () => {
       const element = await page.find("calcite-input-date-picker");
       expect(await element.getProperty("value")).toBe("2020-03-07");
       expect(await element.getProperty("valueAsDate")).toBeDefined();
+
+      // emit when date cleared
+      await input.press("Backspace");
+      await input.press("Backspace");
+      await input.press("Backspace");
+      await input.press("Backspace");
+      await page.waitForChanges();
+      expect(changeEvent).toHaveReceivedEventTimes(8);
+      expect(deprecatedChangeEvent).toHaveReceivedEventTimes(8);
+
+      expect(await element.getProperty("value")).toBe("");
+      expect(await element.getProperty("valueAsDate")).toBeUndefined();
     });
```

Comment:
Can you also add tests that cover no emitting when clearing the value programmatically?